setup.py

```diff
@@ -13,25 +13,35 @@
 # limitations under the License.
 
 from distutils import spawn
-import subprocess
+import importlib
 import os
+import subprocess
 import sys
 
 from setuptools import setup, find_packages
 
+project_name = 'jax'
+
 _current_jaxlib_version = '0.4.15'
 # The following should be updated with each new jaxlib release.
 _latest_jaxlib_version_on_pypi = '0.4.15'
 _available_cuda11_cudnn_versions = ['86']
 _default_cuda11_cudnn_version = '86'
 _default_cuda12_cudnn_version = '89'
 _libtpu_version = '0.1.dev20230830'
-
-_dct = {}
-with open('jax/version.py', encoding='utf-8') as f:
-  exec(f.read(), _dct)
-__version__ = _dct['__version__']
-_minimum_jaxlib_version = _dct['_minimum_jaxlib_version']
+_minimum_jaxlib_version = "0.4.14"
```

Comment:
It's slightly unfortunate this version number is now duplicated in two places.

Reply:
Oh, this was leftover from a previous change. Deleted